internal/kafka/internal/services/kafka.go

```diff
@@ -966,6 +966,7 @@ type KafkaStreamingUnitCountPerRegion struct {
 	ClusterId     string
 	Count         float64
 	CloudProvider string
+	MaxUnits      float64
```

Comment:
Good point. This was just my attempting to avoid cascading casting all over as the metrics needs a `float64`. It is the same type for the `Count` field. Do you think we can make them as int at metric level?

Reply:
that looks good to me :) Thanks!